snapcraft/extensions/kde_neon_6.py

```diff
@@ -87,8 +93,11 @@ def is_experimental(base: Optional[str]) -> bool:
 
     @overrides
     def get_app_snippet(self) -> Dict[str, Any]:
+        command_chain = ["snap/command-chain/desktop-launch"]
+        if self.yaml_data["base"] == "core24":
+            command_chain.insert(0, "snap/command-chain/gpu-2404-wrapper")
         return {
-            "command-chain": ["snap/command-chain/desktop-launch6"],
+            "command-chain": command_chain,
             "plugs": [
                 "desktop",
```

Comment:
Please remove the `network` and `network-bind` plug from [here](https://github.com/canonical/snapcraft/pull/5092/files#diff-333401a9c3bc7c7c35d0eb9875f579c9daf923fee2ded5948762781d62cc30aeR109-R110). I have checked with most of the snaps in KDE, they do have access to network which need them. This is an one time curation.